### ISSUE TITLE
core 3.1.2 does not use correct schema when sending alarm suspend message.

### ISSUE THREAD

**Issue by sveitech** [rsmp-nordic/rsmp_validator#445]
The validator sends this message:

```
Sent Alarm  {"mType":"rSMsg","type":"Alarm","ntsOId":"","xNId":"","xACId":"","xNACId":"","aSp":"Resume","mId":"6d56d25b-73af-4eb1-bffd-5dfad9c0bbb5","cId":"AA+BBCCC=DDDDL002","aCId":"A0302"}
```

Notice the "aSp" field, which reads "Resume". According the the docs (https://rsmp-nordic.github.io/rsmp_core/3.1.2/applicability/usage_of_json.html#alarm-messages) the "aSp" field is lower-case for core 3.1.2. The particular case for "resume" is not documented, but the other shown cases are lower-case.

**Comment by emiltin** [comment]
Right, for core 3.1.2 and 3.1.3, figure 4.8 shows "suspend" and "resume" for aSp:
https://rsmp-nordic.github.io/rsmp_core/3.1.3/applicability/basic_structure.html#alarm-messages
https://rsmp-nordic.github.io/rsmp_core/3.1.3/applicability/basic_structure.html#alarm-messages

From core 3.1.4, figure 4.4. shows "Resume" and "Suspend" for aSp:
¨https://rsmp-nordic.github.io/rsmp_core/3.1.4/applicability/basic_structure.html#alarm-messages

So it seems lowercase "suspend" and "resume" should be used for core <= 3.1.3.

**Comment by emiltin** [comment]
transfering to the schema repo at https://github.com/rsmp-nordic/rsmp_schema/issues
